rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -209,8 +209,13 @@ void gui_application::InitializeCallbacks()
 void gui_application::StartPlaytime()
 {
 	const QString serial = qstr(Emu.GetTitleID());
-	m_gui_settings->SetLastPlayed(serial, QDate::currentDate().toString("MMMM d yyyy"));
-	m_timer_playtime.start();
+	const auto isPUPFirmware = serial.isEmpty();
```

Comment:
Just return here if serial.isEmpty().
There is also no need to reference the firmware here, since empty serials shouldn't be tracked anyway.
```suggestion
	if (serial.isEmpty())
	{
		return;
	}
```

Reply:
Thanks for the review.
My preference is less return statement but I will adapt